### ISSUE TITLE
setDefaultPersistent() wrapping content with " "

### ISSUE THREAD

**Issue by rslonik** [okgrow/meteor-persistent-session#31]
```
setDefaultPersistent( 'id', Random.id() )
console.log( Session.get( 'id' ) );
```

Open incognito window, see console, then refresh.

First time returns zXyRASRxc3HaFTxhm
But after refresh, return changes: "zXyRASRxc3HaFTxhm"

**Comment by rgould** [comment]
Thanks for the bug reports. I'll look into them as soon as I can!

**Comment by rgould** [comment]
@rslonik I wasn't ever able to replicate this, but it may be fixed in the new version (0.4.0). Can you confirm if this is still an issue?

**Comment by rgould** [comment]
@rslonik Both 0.4.0 and 0.4.2 should have fixes that would stop this from happening, so I'll close this issue. If you're still encountering it, go ahead and reopen it.
